tests/integ/modin/interoperability/plotly/test_plotly.py

```diff
@@ -0,0 +1,188 @@
+    ).to_plotly_json()
+    assert_plotly_equal(snow_res, native_res)
+
+
+@sql_count_checker(query_count=2)
+def test_plotly_violin(df):
+    snow_res = px.violin(df, y="y").to_plotly_json()
+    native_res = px.violin(df._to_pandas(), y="y").to_plotly_json()
+    assert_plotly_equal(snow_res, native_res)
+
+
+@sql_count_checker(query_count=2)
+def test_plotly_barplot(df):
+    snow_res = px.bar(df, x="category", y="y").to_plotly_json()
+    native_res = px.bar(df._to_pandas(), x="category", y="y").to_plotly_json()
+    assert_plotly_equal(snow_res, native_res)
+
+
+@sql_count_checker(query_count=2)
+def test_plotly_histogram(df):
+    snow_res = px.histogram(df, x="category").to_plotly_json()
+    native_res = px.histogram(df._to_pandas(), x="category").to_plotly_json()
+    assert_plotly_equal(snow_res, native_res)
+
+
+@sql_count_checker(query_count=2)
+def test_plotly_pie(df):
+    snow_res = px.pie(df, values="category", names="category2").to_plotly_json()
+    native_res = px.pie(
+        df._to_pandas(), values="category", names="category2"
+    ).to_plotly_json()
+    assert_plotly_equal(snow_res, native_res)
+
+
+@sql_count_checker(query_count=2)
+def test_plotly_treemap(df):
+    snow_res = px.treemap(df, names="category", values="y").to_plotly_json()
+    native_res = px.treemap(
+        df._to_pandas(), names="category", values="y"
+    ).to_plotly_json()
+    assert_plotly_equal(snow_res, native_res)
+
+
+@sql_count_checker(query_count=2)
+def test_plotly_sunburst(df):
+    snow_res = px.sunburst(df, names="category", values="y").to_plotly_json()
+    native_res = px.sunburst(
+        df._to_pandas(), names="category", values="y"
+    ).to_plotly_json()
+    assert_plotly_equal(snow_res, native_res)
+
+
+@sql_count_checker(query_count=2)
+def test_plotly_icicle(df):
+    snow_res = px.icicle(df, names="category", values="y").to_plotly_json()
+    native_res = px.icicle(
+        df._to_pandas(), names="category", values="y"
+    ).to_plotly_json()
+    assert_plotly_equal(snow_res, native_res)
+
+
+@sql_count_checker(query_count=2)
+def test_plotly_scatter_matrix(df):
+    snow_res = px.scatter_matrix(df, dimensions=["category"]).to_plotly_json()
+    native_res = px.scatter_matrix(
+        df._to_pandas(), dimensions=["category"]
+    ).to_plotly_json()
+    assert_plotly_equal(snow_res, native_res)
+
+
+@sql_count_checker(query_count=2)
+def test_plotly_funnel(df):
+    snow_res = px.funnel(df, x="x", y="y").to_plotly_json()
+    native_res = px.funnel(df._to_pandas(), x="x", y="y").to_plotly_json()
+    assert_plotly_equal(snow_res, native_res)
+
+
+@sql_count_checker(query_count=2)
+def test_plotly_density_heatmap(df):
+    snow_res = px.density_heatmap(df, x="x", y="y").to_plotly_json()
+    native_res = px.density_heatmap(df._to_pandas(), x="x", y="y").to_plotly_json()
+    assert_plotly_equal(snow_res, native_res)
+
+
+@sql_count_checker(query_count=2)
+def test_plotly_boxplot(df):
```

Comment:
```suggestion
def test_boxplot(df):
```

Reply:
Or: `def test_box(df)`

Reply:
That's true, these should be updated now!